### DISCUSSION TITLE
Ente Auth is displaying blank screen in MX Linux

### DISCUSSION THREAD

**Discussion by pnbalaji** [ente-io/ente#1423] [General]
Hello Team, I have been using Authy for several years. Since Authy has discontinued their desktop apps, I was looking for a better replacement and I see that Ente Auth is just as perfect as Authy. I installed the desktop app of Ente Auth in my MX Linux, but clicking on it just showing a blank screen and nothing else.

Can someone help? Below is a screenshot of what I am seeing.

![image](https://github.com/ente-io/ente/assets/12608966/fc4476de-062c-42f3-b4e9-9b2b457103c6)

**Comment by pnbalaji** [comment]
I ran the authenticator from command line and I am seeing these errors.
```

$ ente_auth 
flutter:  [ente_logging] [INFO] [2024-04-12 15:14:44.813554] log file for today: File: '/home/balashaki/.local/share/ente_auth/logs/2024-4-12.txt' with prefix 
flutter: [ente_logging] [INFO] [2024-04-12 15:14:44.813554] app version: '2.0.54+254'
flutter: [ente_logging] [INFO] [2024-04-12 15:14:44.813554] 
flutter:  [main] [INFO] [2024-04-12 15:14:44.813959] Starting app in foreground
flutter:  [Configuration] [INFO] [2024-04-12 15:14:44.824271] Skipping temp folder clear

** (ente_auth:14702): WARNING **: 15:14:44.847: libsecret_error: Failed to unlock the keyring
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: PlatformException(Libsecret error, Failed to unlock the keyring, null, null)
#0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:651)
#1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:334)
<asynchronous suspension>
```

**Reply by pnbalaji** [reply]
Issue has been resolved. I had to install gnome-keyring package and reboot the PC. Ente Auth works fine now.

**Reply by ppaauuoo** [reply]
thanks! it's work now!
